src/appkit.cs

```diff
@@ -6722,6 +6738,11 @@ interface NSFontDescriptor : NSSecureCoding, NSCopying {
 		[Export ("fontDescriptorWithDesign:")]
 		[return: NullAllowed]
 		NSFontDescriptor Create (NSFontDescriptorSystemDesign design);
+	
+		[Mac (10,16)]
+		[Static]
+		[Export ("preferredFontDescriptorForTextStyle:options:")]
+		NSFontDescriptor PreferredFontDescriptorForTextStyle (string style, NSDictionary options);
```

Comment:
Unless this is an overload of an existing method I would use

```suggestion
		NSFontDescriptor GetPreferredFont (string textStyle, NSDictionary options);
```